list.go

```diff
@@ -216,6 +217,10 @@ func fillPackage(p *build.Package) error {
 	if p.Goroot {
 		return nil
 	}
+	version, err := goVersion()
```

Comment:
I think this should use the recorded go version from the Godeps.json file, not whichever go version is currently running.